Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -42,6 +42,10 @@ struct PDFContextMenu;
 class WebFrame;
 class WebMouseEvent;
 
+enum class AnnotationSearchDirection : uint8_t {
```

Comment:
Can be packed more tightly
```suggestion
enum class AnnotationSearchDirection : bool {
```

Reply:
Done!